src/CMakeLists.txt

```diff
@@ -0,0 +1,209 @@
+add_executable(VNote main.cpp
+    vmainwindow.cpp
```

Comment:
Indent this one properly?

Reply:
This indentation is wrong, it should be fixed.